config/labels.go

```diff
@@ -64,12 +63,20 @@ const LABEL_IS_BACKUP = "is-backup"
 const LABEL_STARTUP = "startup"
 const LABEL_SHUTDOWN = "shutdown"
 
-const LABEL_MINOR_UPGRADE = "minor-upgrade"
-const LABEL_UPGRADE_IN_PROGRESS = "upgrade-in-progress"
-const LABEL_UPGRADE_COMPLETED = "upgrade-complete"
-const LABEL_UPGRADE_REPLICA = "upgrade-replicas"
-const LABEL_UPGRADE_PRIMARY = "upgrade-primary"
-const LABEL_UPGRADE_BACKREST = "upgrade-backrest"
+//label for the pgcluster upgrade
+const LABEL_UPGRADE = "upgrade"
+
+//label indicating the pgcluster requires an upgrade
+const LABEL_IS_UPGRADED = "is-upgraded"
+
+//label showing upgrade from and to information
+const LABEL_UPGRADE_INFO = "upgrade-info"
+
+// label for the pgbadger port that can be lost during an upgrade
+const LABEL_PGBADGER_PORT = "pgbadger-port"
+
+// label for the exporter port that can be lost during an upgrade
+const LABEL_EXPORTER_PORT = "exporter-port"
```

Comment:
Same as above, i.e. isn't this stored under `exporterport` in the spec?

Reply:
same issue as for pgbadger port, please see that description.

Reply:
updated to match the behavior used for the pgbadger port, as described above. Thanks!